src/System.IO.Compression/src/Interop/Interop.clrcompression.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.IO.Compression;
+
+internal static partial class Interop
+{
+    internal static partial class clrcompression
```

Comment:
Can we rename this to zlib?  Even if the binary name for Windows is clrcompression.dll, I think it'd be nice for this to match the actual library name.